pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_Ecker2015/graphite_electrolyte_exchange_current_density_Ecker2015.py

```diff
@@ -33,7 +33,7 @@ def graphite_electrolyte_exchange_current_density_Ecker2015(c_e, c_s_surf, T):
         Exchange-current density [A.m-2]
     """
 
-    k_ref = 1.995 * 1e-10
+    k_ref = 1.11 * 1e-10
```

Comment:
is this explicitly in the paper, or calculated by you?

Reply:
Calculated by me. What's in the paper is the exchange curent density at 50% lithiation.

The reason the value is out is it was changed to 1.995E-10 to account for not all of the solid phase being active. But when the provision for inactive material was added, the rate constants weren't changed back. @rtimms argued the model isn't sensitive enough to this parameter to justify changing, and for a discharge curve it's not, but we have GITT results for this cell that are massively improved by changing the rate constants.

Reply:
Not sure I remember this. Would be keen to see the GITT results if you’re able to share. Even though changes like this make a better fit I do worry a bit about making them in this way. People may rightly expect the params from the Ecker2015 set to be exactly those in the paper, not a tweaked version. This is something we should bear in mind. 

I am aware I added thermal params to this set from a different source, so I’m guilty of making additions/tweaks too.

Reply:
I'm not tweaking though. The rate constant can be calculated directly from the exchange current density at any given lithiation, and that is included in the paper.

Reply:
But doesn’t this change the fit given in the paper (fig 11 of part 1)? Or am I misunderstanding something?